modules/scan/certificate_information.yaml

```diff
@@ -0,0 +1,50 @@
+# echo | openssl s_client -showcerts -connect expired.badssl.com:443
+info:
+  id: untrusted_certificate_scan
```

Comment:
best to name all ssl-related modules with **ssl_** prefix for a consistent naming convention, e.g.
certificate_information->ssl_certificate_information 
weak_tls_ssl_cipher_suites -> ssl_weak_tls_cipher_suites , insecure_tls_ssl_cipher_suites -> ssl_insecure_tls_cipher_suites, soon_to_expire_certificate -> ssl_soon_to_expire_certificate , untrusted_certificate  -> ssl_untrusted_certificate